tools/quarantinerepairman.go

```diff
@@ -1,5 +1,10 @@
 package tools
 
+// In /etc/hummingbird/andrewd-server.conf:
+// [quarantine-repairman]
+// initial-delay = 1       # seconds to wait between requests for the first pass
+// pass-time-target = 3600 # seconds to try to make subsequent passes take
```

Comment:
every other config value thing uses "_" not "-"

Reply:
Oops!